bench_util/benches/op_baseline.rs

```diff
@@ -30,11 +30,11 @@ async fn op_pi_async() -> i64 {
 }
 
 fn bench_op_pi_json(b: &mut Bencher) {
-  bench_js_sync(b, r#"Deno.core.opSync("op_pi_json", null);"#, setup);
+  bench_js_sync(b, r#"Deno.core.ops.op_pi_json();"#, setup);
 }
 
 fn bench_op_nop(b: &mut Bencher) {
-  bench_js_sync(b, r#"Deno.core.opSync("op_nop", null, null, null);"#, setup);
```

Comment:
Shouldn't the `null`s be preserved?
Was this measuring argument discarding speed? (lol)

Reply:
I think these are just leftovers from when op call arity was fixed.